pymchelper/input_output.py

```diff
@@ -20,7 +20,15 @@
 
 
 class AggregationType(IntEnum):
-    """Enum for different types of aggregation."""
+    """
+    Enum for different types of aggregation.
+    Below few examples of such aggregation types used in SHIELD-HIT12A:
+      - NoAggregation is used for density (RHO) and material scorer.
```

Comment:
X = x_0 (values are taken from the first job instance)

Reply:
just to be clear, it will be taken from the first file in the input

Reply:
Thats correct, but unfortunately all the files are being read, but their contents being ignored. See
https://github.com/DataMedSci/pymchelper/blob/52947dc09b1cc2af172213102541f243ca132095/pymchelper/averaging.py#L223-L235

Reply:
I will add some comments

Reply:
but that is fine!